data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/NetworkElement.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.crac_impl;
+
+/**
+ * Element of the network in the CRAC file.
+ *
+ * @author Viktor Terruer {@literal <viktor.terrier at rte-france.com>}
```

Comment:
Shouldn't we add the following attributes to this class: list of AbstractRemedialAction, list of Cnec, list of Contingency ?

Reply:
We'll do it later if needed.